ProQuest.js

```diff
@@ -120,79 +120,96 @@ function fetchEmbeddedPdf(url, item, callback) {
 }
 
 function getSearchResults(doc, checkOnly) {
-	var tabs = doc.getElementsByClassName('tabContent');
 	var root;
-	if (tabs.length) {
-		for (var i = 0; i < tabs.length; i++) {
-			if (tabs[i].offsetHeight) {
-				if (Zotero.isBookmarklet && tabs[i].id != 'allResults-content') return false;
-				root = tabs[i].getElementsByClassName('resultListContainer')[0];
+	var tabs = doc.getElementsByClassName('tabContent');
+	for (var i = 0; i < tabs.length; i++) {
+		if (tabs[i].offsetHeight) {
+			if (Zotero.isBookmarklet && tabs[i].id != 'allResults-content') return false;
+			root = tabs[i].getElementsByClassName('resultListContainer')[0];
+			break;
+		}
+	}
+	if (!root) {
+		var elements = doc.getElementsByClassName('resultListContainer');
+		for (var i=0; i<elements.length; i++) {
+			if (elements[i] && elements[i].offsetHeight>0) {
```

Comment:
Of course

On Tue, Aug 25, 2015 at 11:11 AM, Philipp Zumstein <notifications@github.com
> wrote:

> In ProQuest.js
> <https://github.com/zotero/translators/pull/940#discussion_r37884588>:
>
> > -		for (var i = 0; i < tabs.length; i++) {
> > -			if (tabs[i].offsetHeight) {
> > -				if (Zotero.isBookmarklet && tabs[i].id != 'allResults-content') return false;
> > -				root = tabs[i].getElementsByClassName('resultListContainer')[0];
> > +	var tabs = doc.getElementsByClassName('tabContent');
> > +	for (var i = 0; i < tabs.length; i++) {
> > +		if (tabs[i].offsetHeight) {
> > +			if (Zotero.isBookmarklet && tabs[i].id != 'allResults-content') return false;
> > +			root = tabs[i].getElementsByClassName('resultListContainer')[0];
> > +			break;
> > +		}
> > +	}
> > +	if (!root) {
> > +		var elements = doc.getElementsByClassName('resultListContainer');
> > +		for (var i=0; i<elements.length; i++) {
> > +			if (elements[i] && elements[i].offsetHeight>0) {
>
> I don't follow here what you are writing. Is it okay, if you make the
> changes during the merge? I will then just write a comment //aurimasv
> will updated here for you.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zotero/translators/pull/940/files#r37884588>.
>